docs/animated-switch-navigator.md

```diff
@@ -0,0 +1,19 @@
+---
+id: animated-switch-navigator
+title: createAnimatedSwitchNavigator
+sidebar_label: createAnimatedSwitchNavigator
+---
+
+An SwitchNavigator with animation support. 
+
+To use this navigator, you need to install `react-native-reanimated >= 1.0.0`. If you have a managed Expo project, you need to use >= SDK 33 to have the correct version of Reanimated.
+
+## API Definition
+
+```js
+import createAnimatedSwitchNavigator from 'react-navigation-animated-switch';
+
+createAnimatedSwitchNavigator(RouteConfigs, SwitchNavigatorConfig);
+```
+
+The route configs and navigator configs are identical to [createSwitchNavigator](switch-navigator.html)
```

Comment:
not exactly true, you a `transition` option as described in https://github.com/react-navigation/react-navigation-animated-switch#usage